examples/mensuration_and_polling/grab_georeferenced_image.py

```diff
@@ -0,0 +1,87 @@
+from sentinelhub import (
```

Comment:
please add docstrings to each function explaining what it is suppose to be doing in this example.
also - are we sure all 3rd party dependencies are provided along the scripts? if not, please add installation instructions

Reply:
good point, will do this

Reply:
done